manufacturing-ai-suite/industrial-edge-insights-time-series/apps/weld-anomaly-detection/telegraf-config/Telegraf_devmode.conf

```diff
@@ -3942,9 +3942,9 @@
 #
 #   ## Topics to subscribe to
     topics = [
-     "wind_turbine_data",
+     "weld-data",
      ]
-    name_override = "wind_turbine_data"
+    name_override = "weld-data"
```

Comment:
Hey @pooja-intel ,
As discussed, can we take off the Telegraf_devmode.conf and any other such devmode files?

You can delete the Telegraf.conf and rename the Telegraf_devmode.conf as Telegraf.conf file whatever seems to be appropriate.